courses/machine_learning/deepdive2/time_series_prediction/4_modeling_keras.ipynb

```diff
@@ -0,0 +1,1261 @@
+{
```

Comment:
<p>We are not using the features that we made in my lab for feature engineering. </p><p> </p><p>close_MIN_prior_5_daysclose_MIN_prior_20_daysclose_MIN_prior_260_daysclose_MAX_prior_5_daysclose_MAX_prior_20_daysclose_MAX_prior_260_daysclose_AVG_prior_5_daysclose_AVG_prior_20_daysclose_AVG_prior_260_daysclose_STDDEV_prior_5_daysclose_STDDEV_prior_20_daysclose_STDDEV_prior_260_days</p><p> </p><p>can you make a bonus that says for them to try incorporating these features and see if it boosts model performance?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/689/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='74'/>

Reply:
<p>done</p>